### PR TITLE
OCPBUGS-1748: PipelineRun templates must be fetched from OpenShift namespace

### PR DESCRIPTION
**Pull request by Lucifergene**
**Fixes**: 

https://issues.redhat.com/browse/OCPBUGS-1748

**Analysis / Root cause**: 

PipelineRun templates are currently fetched from `openshift-pipelines` namespace. It has to be fetched from `openshift` namespace.
Align with operator changes https://issues.redhat.com/browse/SRVKP-2413 in 1.8.1, UI has to update the code to fetch pipelinerun templates from openshift namespace.

**Solution Description**: 

The corresponding namespace is updated in the UI.

**Screen shots / Gifs for design review**: 

plr

**Unit test coverage report**: 

No changes.

**Test setup:**

1. Open Admin Perspective
2. Go to Pipelines section.
3. Click on the PipelineRun Tab
4. Click on "Setup Github App" button.
5. Note the disabled project namespace.

**Browser conformance**: 

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by karthikjeeyar** [comment]
lets wait for 1.8.1 release to be available in latest channel

/hold

**Comment by invincibleJai** [comment]
@Lucifergene you would need to  update tests as well https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/12235/pull-ci-openshift-console-master-frontend/1588435783683936256

frontend job is failing

**Comment by karthikjeeyar** [comment]
unholding the PR as 1.8.1 is released

/unhold

File: frontend/packages/pipelines-plugin/src/components/pipelines/const.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const DEFAULT_SAMPLES = 60;
 // Annotation for referencing pipeline name in case of PipelineRun with no reference to a Pipeline (embedded pipeline)
 export const preferredNameAnnotation = 'pipeline.openshift.io/preferredName';
 
-export const PIPELINE_NAMESPACE = 'openshift-pipelines';
+export const PIPELINE_NAMESPACE = 'openshift';
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/src/components/pipelines/const.ts:69]: We should change the utils which is fetching the pipeline run templates, PIPELINE_NAMESPACE should be `openshift-pipelines`.

Comment (karthikjeeyar) [reply frontend/packages/pipelines-plugin/src/components/pipelines/const.ts:69]: Bumping this thread again Incase if you have missed my previous comment, this `PIPELINE_NAMESPACE `  const is used in lot of other places so please dont modify this const, you can update the namespace in the method where it fetches the pipelinerun templates.

Comment (jerolimov) [reply frontend/packages/pipelines-plugin/src/components/pipelines/const.ts:69]: @Lucifergene can you address this comment and rebase your PR?

Comment (karthikjeeyar) [reply frontend/packages/pipelines-plugin/src/components/pipelines/const.ts:69]: From 1.8.1 onwards the templates will be moved to openshift namespace, we should fix this and backport it to 4.11 as mentioned in the ticket.

**Comment by Lucifergene** [comment]
This PR got auto-closed since I pushed in from a different branch from my local system!
The work has been continued in PR #12537.

**Pull request closed without merging**